### DISCUSSION TITLE
Continuação do curso?

### DISCUSSION THREAD

**Discussion by Tmaturano** [balta-io/2811#19] [Q&A] [answered]
Bom dia Balta.
No último módulo, foi comentado algo sobre CI/CD e publicação no Azure. Este curso será incrementado para ter esses dois tópicos?

**Comment by andrebaltieri** [comment, marked as answer]
Sim @Tmaturano, está em produção... em breve teremos novidades <3

**Reply by Tmaturano** [reply]
Muitíssimo obrigado :)
